README.md

```diff
@@ -40,6 +40,13 @@ The addons-linter will check your add-on and show you errors, warnings, and frie
 addons-linter --help
 ```
 
+#### Privileged extensions
+
+The addons-linter can lint **privileged extensions** using the `--privileged` CLI flag. This option changes the behavior of the linter to:
+
+1. emit errors when the input file (or directory) is a regular extension (i.e. the extension does not use privileged features)
```

Comment:
Nit, Maybe "the extension does not require any privileged feature through its manifest.json file"? just to be clear that it is what is in declared are required in the manifest to make a difference here).